DataAccessGeneration/Generator.cs

```diff
@@ -657,6 +659,32 @@ private static string AddShorthandMethod(string procName, List<ParameterDefiniti
 
         return "";
     }
+    private static string AddUDTShorthandMethod(string procName, List<ParameterDefinition> parameters, List<UserDefinedTableRowDefinition> userDefinedTypes, string methodReturnType, ResultMetaData resultMetaData)
+    {
+        StringBuilder sb = new StringBuilder();
+        if (parameters.Count != 1) return "";
```

Comment:
do you need to return a value if it's going to be empty or can you just `return`

Reply:
I was just having it show an empty string (which will later get removed with line breaks). Just thought it would be easier to generate that way. Note the way it is called inside of a code representation. The string builder no longer makes sense and has been removed